Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+pipeline {
+  agent any
+  environment {
+    GIT = credentials('github')
+  }
+  stages {
+    stage('Prerequisites') {
+      steps {
+        sh '/usr/local/bin/pod install --project-directory=Example/'
```

Comment:
Instead of using the full path, can you use an environment? Like here:
https://github.com/gini/gini-switch-sdk-android/blob/develop/Jenkinsfile#L65

Reply:
I saw it, but for me setting the absolute path is more clear.